.github/workflows/fast_testing.yml

```diff
@@ -10,6 +10,40 @@ on:
       - '*'
 
 jobs:
+  version_check:
```

Comment:
Release publish should depend on this one.

Reply:
We have code duplication now.

Reply:
Just remove it

Reply:
Done, now there is only in `publish.yml`